docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Intel Platform Software for Windows (PSW) v2.7](http://registrationcenter-download.intel.com/akdlm/irc_nas/16464/Intel%20SGX%20PSW%20for%20Windows%20v2.7.100.2.exe)
 
+The latest PSW should be installed automatically on a Windows machine with Windows
+version no lower than 1709. To check your Windows version, run `winver` on the
+command line.
```

Comment:
Consider rephrasing this for clarity:
``` suggestion
The PSW only needs to be manually installed if you are running on Windows Server
2016 or a version of Windows client lower than 1709. It should be installed automatically
on newer versions of Windows client and Windows Server 2019. You can check your
version of Windows by running `winver` on the command line.
```

Nonblocking: One of our users asked the question around what they should do if they don't enable Windows updates on their devices, and therefore may not automatically be getting the latest versions of the PSW automatically installed. 

Since this document is explicitly about manually installing dependencies, and we're adding WS2019 support, this may be a good time to add documentation around manually updating the drivers and PSW components with the versions provided under the `PSW_INF_RS3_and_above` folder as well.

This suggestion also applies to WindowsManualSGX1Prereqs.md.

Reply:
I would also recommend mentioning that it is installed via Windows Update in case things like GPO policy are blocking WU.